modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/method_init_normal.mustache

```diff
@@ -8,12 +8,6 @@
 
 {{> python-experimental/model_templates/method_init_shared }}
 
-        self._data_store = {}
-        self._check_type = _check_type
-        self._from_server = _from_server
-        self._path_to_item = _path_to_item
-        self._configuration = _configuration
-
 {{#requiredVars}}
         self.{{name}} = {{name}}
 {{/requiredVars}}
```

Comment:
How about adding this if statement in the below for loop?
```
            if var_name not in self.attribute_map and self._configuration is not None and \
                            self._configuration.discard_unknown_keys and \
                            self.additional_properties_type is None:
                # discard variable.
                pass
```
That will let us skip unknown keys for these non composed schema models.

Reply:
done

> How about adding this if statement in the below for loop?
> 
> ```
>             if var_name not in self.attribute_map and self._configuration is not None and \
>                             self._configuration.discard_unknown_keys and \
>                             self.additional_properties_type is None:
>                 # discard variable.
>                 pass
> ```
> 
> That will let us skip unknown keys for these non composed schema models.

done